.github/workflows/ui_test.yml

```diff
@@ -0,0 +1,63 @@
+name: Testing workflow
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: PostgreSQL
+        uses: ikalnytskyi/action-setup-postgres@v5
+
+      - name: Postgres setup
+        env:
+          PGHOST: localhost
+          PGDATABASE: evershop
+          PGUSER: postgres
+          PGPASSWORD: postgres
+        run: |
+          createdb --owner $PGUSER $PGDATABASE
+          psql -c "ALTER USER $PGUSER WITH PASSWORD '$PGPASSWORD'"
```

Comment:
Please, have a look at the `services`. This would be nice place for db services
https://docs.github.com/en/actions/using-containerized-services/creating-postgresql-service-containers#running-jobs-in-containers

Reply:
okay